.github/workflows/ci.yml

```diff
@@ -50,11 +50,29 @@ jobs:
         # temporarily run without TSAN due to false positives on Linux
         # https://github.com/SwiftPackageIndex/SwiftPackageIndex-Server/pull/1496
         # Review with Swift 5.6
-        run: cp .env.testing.template .env.testing && make test-without-tsan
+        run: |
+          # setup environments
+          echo "
+          DATABASE_HOST=postgres
+          DATABASE_NAME=spi_test
+          DATABASE_PASSWORD=xxx
+          DATABASE_PORT=5432
+          DATABASE_USERNAME=spi_test
+          " >> .env.testing
+
+          echo "
+          DATABASE_HOST=${{ secrets.STAGING_DATABASE_HOST }}
+          DATABASE_NAME=spi_dev
+          DATABASE_PASSWORD=${{ secrets.STAGING_DATABASE_PASSWORD }}
+          DATABASE_PORT=5432
+          DATABASE_USE_TLS=true
+          DATABASE_USERNAME=spi_dev
+          " >> .env.staging
+
+          make test-without-tsan
+          make test-query-performance
```

Comment:
Instead of running the two tests back to back we could also add another job specifically to run the query performance tests. I've put them together here because build time is ~8 mins vs test time of ~1 min.

I think if we decide to do this, we _should_ set up another job and schedule it to run daily only, in which case the extra build time doesn't matter really.

Reply:
I'll clean this up once we decide to go forward and restore the original job to what it was before.